specs/eventing/control-plane.md

```diff
@@ -0,0 +1,985 @@
+   `backoffPolicy` and `backoffDelay` parameters if specified. Alternatively,
+   implementations MAY use an equivalent internal mechanism for delivery (for
+   example, if the `ref` form of `deadLetterSink` points to a compatible
+   implementation).
+
+## Detailed Resources
+
+### Broker v1
+
+#### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource.
+
+#### Spec:
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>config</code></td>
+    <td><a href="#kreference">KReference</a><br/>(Optional)</td>
+    <td>A reference to an object which describes the configuration options for the Broker (for example, a ConfigMap).</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>delivery</code></td>
+    <td><a href="#deliveryspec">DeliverySpec</a><br/>(Optional)</td>
+    <td>A default delivery options for Triggers which do not specify more-specific options.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+#### Status
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>conditions</code></td>
+    <td><a href="#error-signalling">See Error Signalling</a></td>
+    <td>Used for signalling errors, see link. Conditions of types Ready MUST be present.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>observedGeneration</code></td>
+    <td>int64</td>
+    <td>The latest metadata.generation that the reconciler has attempted. If <code>observedGeneration</code> is updated, <code>conditions</code> MUST be updated with current status in the same transaction.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>address</code></td>
+    <td><a href="#duckv1addressable">duckv1.Addressable</a></td>
+    <td>Address used to deliver events to the Broker.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>deadLetterSinkUri</td>
+    <td>URL (string)</td>
+    <td>If <code>spec.delivery.deadLetterSink</code> is specified, the resolved URL of the dead letter address.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### Trigger v1
+
+#### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource.
+
+#### Spec:
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>broker</code></td>
+    <td>string<br/>(Required, Immutable)</td>
+    <td>The Broker which this Trigger should filter events from.</td>
```

Comment:
It might be better to just say "Broker to which this Trigger is associated", then we don't need the SHOULD since that's already covered in other text. But if you leave it, then why is it a "should" and not a "MUST" ?